docs/release-info/release-notes.md

```diff
@@ -73,47 +132,208 @@ This section contains detailed release notes separated by areas.
+
+#### Segment metadata cache metrics
+
+The following metrics have been added:
+
+The following metrics have been introduced as part of the segment metadata cache performance improvement.
+
+- `segment/used/count`
+- `segment/unused/count`
+- `segment/pending/count`
+- `segment/metadataCache/sync/time`
+- `segment/metadataCache/deleted`
+- `segment/metadataCache/skipped`
+- `segment/metadataCache/used/stale`
+- `segment/metadataCache/used/updated`
+- `segment/metadataCache/unused/updated`
+- `segment/metadataCache/pending/deleted`
+- `segment/metadataCache/pending/updated`
+- `segment/metadataCache/pending/skipped`
+
+For more information about the segment metadata cache, see [Faster segment metadata operations](#faster-segment-metadata-operations).
+
+[#17653](https://github.com/apache/druid/pull/17653)
+
+#### Streaming ingestion lag metrics
+
+The Kafka supervisor now includes additional lag metrics for how many minutes of data Druid is behind:
+
+|Metric|Description|Default value|
+|-|-|
+|`ingest/kafka/updateOffsets/time`|Total time (in milliseconds) taken to fetch the latest offsets from Kafka stream and the ingestion tasks.|`dataSource`, `taskId`, `taskType`, `groupId`, `tags`|Generally a few seconds at most.|
+|`ingest/kafka/lag/time`|Total lag time in milliseconds between the current message sequence number consumed by the Kafka indexing tasks and latest sequence number in Kafka across all shards. Minimum emission period for this metric is a minute. Enabled only when `pusblishLagTime` is set to true on supervisor config.|`dataSource`, `stream`, `tags`|Greater than 0, up to max kafka retention period in milliseconds. |
+|`ingest/kafka/maxLag/time`|Max lag time in milliseconds between the current message sequence number consumed by the Kafka indexing tasks and latest sequence number in Kafka across all shards. Minimum emission period for this metric is a minute. Enabled only when `pusblishLagTime` is set to true on supervisor config.|`dataSource`, `stream`, `tags`|Greater than 0, up to max kafka retention period in milliseconds. |
+|`ingest/kafka/avgLag/time`|Average lag time in milliseconds between the current message sequence number consumed by the Kafka indexing tasks and latest sequence number in Kafka across all shards. Minimum emission period for this metric is a minute. Enabled only when `pusblishLagTime` is set to true on supervisor config.|`dataSource`, `stream`, `tags`|Greater than 0, up to max kafka retention period in milliseconds. |
+|`ingest/kinesis/updateOffsets/time`|Total time (in milliseconds) taken to fetch the latest offsets from Kafka stream and the ingestion tasks.|`dataSource`, `taskId`, `taskType`, `groupId`, `tags`|Generally a few seconds at most.|
+
+[#17735](https://github.com/apache/druid/pull/17735)
+
+#### Other metrics and monitoring changes
+
+- Added the `segment/metadataCache/transactions` that describes the number of read or write transactions performed on the cache for a single datasource
```

Comment:
This can be removed from here. Please see other comment regarding the segment metadata cache metrics.